client/src/app/core/repositories/motions/motion-workflow-repository.service.ts

```diff
@@ -87,4 +91,16 @@ export class MotionWorkflowRepositoryService extends BaseRepositoryWithActiveMee
     public delete(viewModel: ViewMotionWorkflow): Promise<void> {
         return this.sendActionToBackend(MotionWorkflowAction.DELETE, { id: viewModel.id });
     }
+
+    public getRequestToGetAllModels(): SimplifiedModelRequest {
+        return {
+            viewModelCtor: ViewMeeting,
+            ids: [this.activeMeetingIdService.meetingId],
+            follow: [
```

Comment:
As a general pattern: Maybe use the fieldsets. Make a new standard fieldset for this method. The advandage is that all fieldsets are declared in one place, so there is just one place to look it up.